avocado/utils/pci.py

```diff
@@ -141,8 +141,7 @@ def get_slot_from_sysfs(full_pci_address):
     if not os.path.isfile("/proc/device-tree/%s/ibm,loc-code" % devspec):
         return
     slot = read_file("/proc/device-tree/%s/ibm,loc-code" % devspec)
-    return re.match(r'((\w+)[\.])+(\w+)-(\w*\d+)-(\w*\d+)|Slot(\d+)',
-                    slot).group()
```

Comment:
sorry for minimal info in commit message.
Different system giving the slot numbers in different ways. like as follows,
root@ltciofvtr-firestone1:~# lsvpd -l 0000:01:00.1
*FC 
*DS fibre-channel (141000e3)
*AX 0000:01:00.1
*AX fibre-channel
*MF IBM
*TM Lancer Gen6: LPe32000 Fibre Channel Host Adapter
*CD 1014e300
*YC 1
*YL Slot5
root@ltciofvtr-firestone1:~# 

root@ltc-briggs3:~# lsvpd -l 0004:01:00.1
*FC 
*DS fibre-channel (77103225)
*AX 0004:01:00.1
*AX fibre-channel
*MF QLogic Corp.
*TM ISP2532-based 8Gb Fibre Channel to PCI Express HBA
*CD 10772532
*YC 2
*YL WIO Slot3
root@ltc-briggs3:~# 

root@ltcalpine-lp1:~# lsvpd -l 0019:01:00.0
*FC 
*DS fibre-channel (14103225)
*AX 0019:01:00.0
*AX fibre-channel
*MF IBM
*TM ISP2532-based 8Gb Fibre Channel to PCI Express HBA
*CD 10142532
*YC 2
*YL U78C7.001.RCH0040-P1-C12-T1
root@ltcalpine-lp1:~# 

*YL filed shows the slot number. all above 3 different servers showed different entry. we may expect even different for upcoming new system. so thought directly giving the file output will make sense. end users can play in their script in the way they need.